Makefile

```diff
@@ -272,7 +272,7 @@ test\:coverage-html:
 .PHONY: test\:changes
 test\:changes:
 	@$(MAKE) deps:lens
-	env DEFRA_DETECT_DATABASE_CHANGES=true gotestsum -- ./... -shuffle=on -p 1
+	env DEFRA_DETECT_DATABASE_CHANGES=true DEFRA_TARGET_BRANCH=1672-assert-true-chd-db-close gotestsum -- ./... -shuffle=on -p 1
```

Comment:
question: Why do we need to set the target branch?

Reply:
For testing, please see the commits.